### PR TITLE
Fixed GPU usage on Windows

### PR DESCRIPTION
**Pull request by Innixma**
*Issue #, if available:*
#1127

*Description of changes:*

- Hopefully fixes issues with GPU not being detected on Windows.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by szha** [comment]
Job PR-1255-1 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1255/1/index.html

File: vision/src/autogluon/vision/predictor/predictor.py [hunk 1]
```diff
@@ -699,6 +699,17 @@ def list_models(cls):
...
+    @staticmethod
+    def _get_num_gpus_available():
+        # FIXME: update to use only torch for TIMM or find a better GPU detection strategy
+        # FIXME: get_gpu_count by itself doesn't always work for Windows
+        num_gpus = get_gpu_count()
+        if num_gpus == 0:
+            num_gpus = get_gpu_count_mxnet()
+            if num_gpus == 0:
+                num_gpus = get_gpu_count_torch()
+        return num_gpus
```
Comment (gradientsky) [inline vision/src/autogluon/vision/predictor/predictor.py:711]: Duplicated code - can we move this into some utility class?

Comment (Innixma) [reply vision/src/autogluon/vision/predictor/predictor.py]: good point, updated

**Review by gradientsky — APPROVED** [review]

**Comment by szha** [comment]
Job PR-1255-3 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1255/3/index.html

**Pull request merged**